.github/workflows/gradle.yml

```diff
@@ -17,7 +17,6 @@ jobs:
     - uses: actions/checkout@v4
       with:
         submodules: true
-        token: ${{ secrets.PAT_TOKEN }}
```

Comment:
This is required - otherwise, GH Actions seems to be unable to get the submodules, from previous experience with this repo.
Not entirely sure why, but it's needed.